test/run-drun/ok/query.drun-run.ok

```diff
@@ -9,11 +9,11 @@ ingress Completed: Reply: 0x4449444c0000
 debug.print: 4
 ingress Completed: Reply: 0x4449444c0000
 ingress Completed: Reply: 0x4449444c00017d04
-debug.print: 5
-Ok: Reply: 0x4449444c00017d04
 debug.print: 4
-ingress Completed: Reply: 0x4449444c0000
 debug.print: 5
-Ok: Reply: 0x4449444c00017d04
+ingress Completed: Reply: 0x4449444c0000
 debug.print: 4
+debug.print: 5
```

Comment:
Hm, also here, the order changed. Is this because of fire-and-forget calls?

Reply:
**Update**: This is due to a race condition in `drun`. To be fixed by https://github.com/dfinity/ic/pull/552.